### PR TITLE
Make Turn Documentation more accessible and add localization

### LINKED ISSUE #38: Make Turn Documentation more accessible
Players will often access the draft through a direct link. The only proper documentation of how turns work however is only available in the preset editor currently.

Make documentation of how exactly Exclusive, Nonexclusive and Global Bans and Picks work, as well as Snipes and Admin Actions, available on the drafting screen. Somehow.

### LINKED ISSUE #39: Localise Preset editor
Currently, the Preset editor is english only.

Use the i18next framework that we use everywhere else to localise that one as well.

### PR DESCRIPTION
**Pull request by lalitpatel**
### Summary of changes:
* Added a section "How it works" in the draft spectate screen (Fixes #38)
* Also added the same info in a separate tab
* Removed the 'Practice' tab
* Localize the Preset Editor (Fixes #39) 

### Screenshots:

tab

collapsed

expanded

### CODE REVIEW

**Comment by HSZemi** [comment]
Do we want to make all the stuff translateable with this merge request as well?

**Comment by lalitpatel** [comment]
Yes, that is a good idea.

File: src/languages/de_DE.json [hunk 1]
```diff
@@ -157,7 +157,10 @@
     "spectate": "Zusehen",
     "recentDraftsTitle": "Die letzten Drafts",
     "watchLive": "Live Zusehen",
-    "watch": "Ansehen"
+    "watch": "Ansehen",
+    "host": "Host",
+    "guest": "Gast",
+    "draftName": "Draft Namen"
```
Comment (HSZemi) [inline src/languages/de_DE.json:163]:
```suggestion
    "draftName": "Draft-Name"
```

File: src/components/draft/Draft.tsx [hunk 2]
```diff
@@ -139,9 +141,18 @@ class Draft extends React.Component<IProps, IState> {
                     </div>
 
                     <CivGrid civilisations={this.props.preset.civilisations}/>
+                </div>
+            </section>
 
+            <section className="section">
+                <div className="container is-desktop">
+                    <details>
+                        <summary><Trans i18nKey='menu.howItWorks'>How it works?</Trans></summary>
```
Comment (HSZemi) [inline src/components/draft/Draft.tsx:150]:
```suggestion
                        <summary><Trans i18nKey='menu.howItWorks'>How it works</Trans></summary>
```

Comment (HSZemi) [reply src/components/draft/Draft.tsx:150]: Can we get a pointer cursor on this? Or would that be considered bad practice?

**Comment by HSZemi** [comment]
grafik
The vertical distance as well as the angle at which it's farther left look really odd to me. Could it be centred, and would that look better?

File: src/languages/en_GB.json [hunk 3]
```diff
@@ -125,16 +125,60 @@
...
+      "4": "You can remove a turn by clicking the <1/> button to the right of the turn."
+    },
+    "presetModifiers": {
+      "title": "Modifiers",
+      "1": "If you want the players to execute two of their turns in parallel, check the <1>PARALLEL</1> checkbox of the <3>first</3> of those two turns (but do not check it for the second turn).",
+      "2": "Check the <1>HIDDEN</1> checkbox to hide the civilisation selected during that turn from the opponent and from spectators until it is revealed by an admin turn (see below)."
+    },
+    "draftModifiers": {
+      "title": "Modifiers",
+      "1": "A <1>PARALLEL</1> modifier means both the players to execute their turns in parallel. The picks or bans are revealed to the other player and spectators only after both the players have made their choices.",
```
Comment (HSZemi) [inline src/languages/en_GB.json:179]: >The picks or bans are revealed to the other player and spectators only after both the players have made their choices.

Whoops, that's a lie :upside_down_face:  
You need to mark them as hidden to achieve that.

**Comment by HSZemi** [comment]
Here are two commits that you can include if you like: https://github.com/SiegeEngineers/aoe2cm2/commits/german-translation-for-how-it-works

File: src/languages/en_GB.json (hunk 3 above)
Comment (lalitpatel) [reply src/languages/en_GB.json:179]: 🙈

**Comment by lalitpatel** [comment]
> grafik
> The vertical distance as well as the angle at which it's farther left look really odd to me. Could it be centred, and would that look better?

The width of the "how it works" section is same as in other pages. I have tried to centre align the text and kept the width of the expanded section same. Since the width of the host and guest boxes is fluid, I feel this is fine. Here is how it looks:

Screen Shot 2020-07-10 at 9 52 57 AM
Screen Shot 2020-07-10 at 9 52 49 AM

**Comment by lalitpatel** [comment]
@HSZemi I have added localization to Preset Editor screen as well. You can add strings for german in `de_DE.json`

**Comment by HSZemi** [comment]
> I have made a few changes and below are the two options. Let me know which one you find better. I find the one which is center aligned more aesthetically pleasing.

I agree.

**Comment by HSZemi** [comment]
Here we go: https://github.com/SiegeEngineers/aoe2cm2/commits/german-translation-for-preset-editor

**Comment by lalitpatel** [comment]
> Here we go: https://github.com/SiegeEngineers/aoe2cm2/commits/german-translation-for-preset-editor

Merged!

**Comment by HSZemi** [comment]
Thanks a lot :yellow_heart:

**Pull request merged**